app/models/form/sales/questions/ownership_scheme.rb

```diff
@@ -0,0 +1,17 @@
+class Form::Sales::Questions::OwnershipScheme < ::Form::Question
+  def initialize(id, hsh, page)
+    super
+    @id = "ownershipsch"
+    @check_answer_label = "Purchase made under ownership scheme"
+    @header = "Was this purchase made through an ownership scheme?"
```

Comment:
Are these strings meant to be in the locale file or is it ok to keep here?

Reply:
Hm, these files mainly contain the definitions and very rarely some specific functionality for a question so they're unlikely to grow large. There are a lot of questions so if we keep all the strings in the locale file it might end up growing very large (which is what we are trying to avoid by putting these questions in separate files). I'd say it makes sense to keep them here